src/Mapbender/CoreBundle/Element/Logout.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+namespace Mapbender\CoreBundle\Element;
+
+use Mapbender\CoreBundle\Component\Element;
+
+class Logout extends Element
+{
+    /**
+     * @return string
+     */
+    public static function getClassTitle()
+    {
+        return "Logout";
+    }
+
+    /**
+     * @return string
+     */
+    public static function getClassDescription()
+    {
+        return "Logout Button!";
+    }
+
+    /**
+     * @return array
+     */
+    public static function getClassTags()
+    {
+        return array("logout");
+    }
+
+    /**
+     * @return array
+     */
+    public static function listAssets()
+    {
+        return array(
+            'js' => array(
+                '@MapbenderCoreBundle/Resources/public/js/mapbender.element.logout.js'
+            ),
+            'css' => array(
+                '@MapbenderCoreBundle/Resources/public/sass/element/logout.scss'
+            ),
+            
+        );
+    }
+
+    /**
+     * @return array
+     */
+    public static function getDefaultConfiguration()
+    {
+        return array(
+            'title'   => 'logout element',
+            'label'   => 'Abmelden',
```

Comment:
Subjects should be translated.
Example: https://github.com/mapbender/mapbender/blob/release/3.0.5/src/Mapbender/ManagerBundle/Controller/ApplicationController.php#L414